EIPS/eip-698.md

```diff
@@ -0,0 +1,98 @@
+## Preamble
+    
+	EIP: 698
+ 
+	Title: OPCODE 0x46 BLOCKREWARD
+    
+	Author: Cody Burns <dontPanic@codywburns.com>
+    
+	Type: Standard Track
+    
+	Category: Core
+    
+	Status: Draft
+    
+	Created: 2017-28-08
+
+
+
+## Simple Summary
+
+This EIP adds an additional opcode to the EVM which will return a finalized blocks reward value. 
+
+
+
+## Abstract
+
+In the EVM, the 0x40 opcodes are reserved for `Block Information`. Currently reserved opcodes are:
+* `0X40 BLOCKHASH`
+* `0X41 COINBASE`
+* `0X42 TIMESTAMP`
+* `0X43 NUMBER`
+* `0X44 DIFFICULTY`
+* `0X45 GASLIMIT`
+
+This EIP would add an additional opcode, `0x46 BLOCKREWARD`, which would return the block reward for any finalized block. The finalized block reward would include the base reward, uncle payments, and gas.   
```

Comment:
I don't think it is possible to get the "final" block reward for the current block since total gas usage cannot be known in advance.